docs/Tutorials/CCE.md

```diff
@@ -0,0 +1,122 @@
+    generally be around 2 lower than the maximum l to reduce possible aliasing).
+  - If you want to just run through the end of the provided worldtube data,
+    you can just omit the `EndTime` option and the executable will figure it
+    out from the worldtube file.
+  - The `ScriOutputDensity` adds extra interpolation points to the output,
+    which is useful for finite-difference derivatives on the output data, but
+    otherwise it'll just unnecessarily inflate the output files, so if you
+    don't need the extra points, best just set it to 1.
+  - If you're extracting at 100M or less, best to reduce the `TargetStepSize`,
+    to around .5 at 100M and lower yet for nearer extraction.
+  - The `InitializeJ` in the example file uses `InverseCubic` which is a pretty
+    primitive scheme, but early tests indicate that it gives the best results
+    for most systems.
+    If initial data is a concern, you can also try replacing the `InverseCubic`
+    entry with:
+    ```
+    NoIncomingRadiation:
+      AngularCoordTolerance: 1.0e-13
+      MaxIterations: 500
+      RequireConvergence: true
+    ```
+    which are probably pretty good choices for those parameters,
+    and the `RequireConvergence: true` will cause the iterative solve in
+    this version to error out if it doesn't find a good frame.
+  .
+- An example of an appropriate submission command for slurm systems is:
+  ```
+  srun -n 1 -c 1 path/to/build/bin/CharacteristicExtract ++ppn 3 \
+ --input-file path/to/input.yaml
+  ```
+  CCE doesn't currently scale to more than 4 cores, so those slurm options are
+  best.
+- CCE will work faster if the input worldtube hdf5 file is chunked in small
+  numbers of complete rows.
+  This is relevant because by default, SpEC writes its worldtube files
+  chunked along full time-series columns, which is efficient for
+  compression, but not for reading in to SpECTRE -- in that case,
+  it is recommended to rechunk the input file before running CCE
+  for maximum performance. This can be done, for instance, using h5py:
+  ```py
+  with h5py.File(input_file,'r') as input_h5,\
+    h5py.File(output_file, 'w') as output_h5:
+    for dset in input_h5:
+        if("Version" in dset):
+            output_h5[dset] = input_h5[dset][()]
+            continue
+        number_of_columns = input_h5[dset][()].shape[1]
+        output_h5.create_dataset(dset, data=input_h5[dset],
+                                 maxshape=(None, number_of_columns),
+                                 chunks=(4, number_of_columns), dtype='d')
+        for attribute in input_h5[dset].attrs.keys():
+            output_h5[dset].attrs[attribute] = input_h5[dset].attrs[attribute]
+  ```
+- The output data will be written as spin-weighted spherical harmonic
+  modes, one physical quantity per dataset, and each row will
+  have the time value followed by the real and imaginary parts
+  of the complex modes in m-varies-fastest order.
+
+### Input data formats
```

Comment:
Our first attempt produced the following error message, that does not enlighten us on what is wrong
```
############ ERROR ############
Node: 0 Proc: 0
Line: 216 of /home/nils/nils/spectre/spectre_nils/src/IO/H5/File.hpp
Function: const ObjectType &h5::H5File<h5::AccessType::ReadOnly>::get(const std::string &, Args &&...) const [Access_t = h5::AccessType::ReadOnly, ObjectType = h5::Dat, Args = <>]
Cannot open the object '/Lapse.dat' because it does not exist.
############ ERROR ############
```

Reply:
the following option was needed
```
H5IsBondiData: True
```
which is not mentioned at all